tools/monorepo-utils/src/slack/README.md

```diff
@@ -0,0 +1,16 @@
+# Slack Utilities
+
+Utilities for automated posting to Slack.
+
+### Usage
+
+This command is built on postinstall and can be run from the root of the project.
+
+From root run `pnpm utils slack --help` to find out more about using the command.
+
+### Development
+
+During development you can watch for changes via `pnpm build --watch` in this directory.
```

Comment:
This would be run from `tools/monorepo-utils` not "this directory". There is also `pnpm start` for this purpose as well.

https://github.com/woocommerce/woocommerce/blob/52a268892b0323db30883d2b94597e33ce5d26a4/tools/monorepo-utils/package.json#L39

Reply:
Ah dang you're right, probably the README shouldn't be in this folder but I should consolidate this info into the top level README

Reply:
@psealock Moved info around in READMEs https://github.com/woocommerce/woocommerce/pull/38185/commits/e55e318493064f84203a114b375c46eba41d7f8f and https://github.com/woocommerce/woocommerce/pull/38185/commits/651a22ee1f29a7deaa54f23edc5def7553dc701e